Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1184,11 +1184,8 @@ WebIDL::ExceptionOr<void> HTMLInputElement::set_size(unsigned value)
 Optional<double> HTMLInputElement::convert_string_to_number(StringView input) const
 {
     // https://html.spec.whatwg.org/multipage/input.html#number-state-(type=number):concept-input-value-string-number
-    if (type_state() == TypeAttributeState::Number) {
-        if (auto number = parse_floating_point_number(input); number.has_value())
-            return *number;
-        return {};
-    }
+    if (type_state() == TypeAttributeState::Number)
+        return parse_floating_point_number(input);
```

Comment:
This fix was squashed into the wrong commit

Reply:
Oops